### ISSUE TITLE
Range of Potamogeton richardsonii in NL

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#2028]
[Originally posted on GoogleCode (id 2258) on 2014-08-03]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/8318

<b>What data are incorrect or missing?</b>
Potamogeton richardsonii is listed from insular Nfld.

<b>What data are you expecting instead?</b>
We had a notation in our NL checklist that says this species is known from specimens from Goose Bay (MT, DAO).  I have no range within Nfld. for this species.1847
Should this have shown a distribution for LB and not Nfld.?

<b>If applicable, please provide an authoritative source.</b>

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2014-08-04 20:01Z]

Need verification
L

**Comment by brouille** [comment]
@sjmeades 
You are correct: no report from Nfld in the MT database (including all data from Rouleau's compilation), and one from Labr.

**Comment by sjmeades** [comment]
Since I originally posted this, I found more information.  Nathalie included the Nfld. report in the 2004 report of the NFRPP.  This is what I've included in the new NL checklist.
[The first report of Potamegeton richardsonii from insular Nfld. comes from a 1997 collection by J.Maunder (NFM) from the shores of Indian Pond, nwNfld. (Djan-Chékar et al. 2004). Two early Labrador reports of Potamogeton richardsonii are from J.M.Gillett & W.I.Findlay, 1950 (DAO, mouth of Hamilton R.) and I.Hustich and P.Kallio, 1963 (CAN, Mud Lake); this species was relocated in the Lower Churchill River in 2006 by M. Sensen (AMEC 2007).]
